connector/keystone/types.go

```diff
@@ -0,0 +1,154 @@
+package keystone
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+const exposedKeystonePort = "5000"
+const exposedKeystonePortAdmin = "35357"
+
+const keystoneHost = "http://localhost:"
+const keystoneURL = keystoneHost + exposedKeystonePort
+const keystoneAdminURL = keystoneHost + exposedKeystonePortAdmin
+const authTokenURL = keystoneURL + "/v3/auth/tokens/"
+const usersURL = keystoneAdminURL + "/v3/users/"
+const groupsURL = keystoneAdminURL + "/v3/groups/"
+
+
+type keystoneConnector struct {
+	Domain 			 string
+	KeystoneHost 	 string
+	KeystoneUsername string
+	KeystonePassword string
+	Logger 			 logrus.FieldLogger
+}
+
+type userKeystone struct {
+	Domain domainKeystone `json:"domain"`
+	ID     string         `json:"id"`
+	Name   string         `json:"name"`
+}
+
+type domainKeystone struct {
+	ID string   `json:"id"`
+	Name string `json:"name"`
+}
+
+// Config holds the configuration parameters for Keystone connector.
```

Comment:
Do you know which (API) version of keystone we're requiring here? There's a few https://developer.openstack.org /api-ref/identity/ 😉

Reply:
We are requiring API version 3. Could you have a second look at this PR ? It seems that CI builds are passing now ;-)